src/Ens/JobeetBundle/Resources/config/doctrine/Category.orm.yml

```diff
@@ -0,0 +1,20 @@
+# src/Ens/JobeetBundle/Resources/config/doctrine/Category.orm.yml
```

Comment:
@PhotonSapiens 
same for all other ORM configurations
And more tmp `.yml~` files here